packages/sit-onyx/src/styles/variables-dark.css

```diff
@@ -0,0 +1,429 @@
+  --onyx-spacing-2xl: var(--onyx-number-spacing-800);
+  --onyx-spacing-3xl: var(--onyx-number-spacing-900);
+  --onyx-size-object-1: var(--onyx-number-size-050);
+  --onyx-size-object-2: var(--onyx-number-size-100);
+  --onyx-size-object-4: var(--onyx-number-size-150);
+  --onyx-size-object-8: var(--onyx-number-size-200);
+  --onyx-size-object-10: var(--onyx-number-size-250);
+  --onyx-size-object-12: var(--onyx-number-size-300);
+  --onyx-size-object-16: var(--onyx-number-size-350);
+  --onyx-size-object-20: var(--onyx-number-size-400);
+  --onyx-size-object-24: var(--onyx-number-size-450);
+  --onyx-size-object-32: var(--onyx-number-size-500);
+  --onyx-size-object-40: var(--onyx-number-size-550);
+  --onyx-size-object-48: var(--onyx-number-size-600);
+  --onyx-size-object-54: var(--onyx-number-size-650);
+  --onyx-size-object-58: var(--onyx-number-size-700);
+  --onyx-size-object-64: var(--onyx-number-size-750);
+  --onyx-size-object-72: var(--onyx-number-size-800);
+  --onyx-size-object-80: var(--onyx-number-size-850);
+  --onyx-size-object-96: var(--onyx-number-size-900);
+  --onyx-size-object-128: var(--onyx-number-size-950);
+  --onyx-size-border-none: 0;
+  --onyx-size-border-sm: var(--onyx-number-size-050);
+  --onyx-size-border-md: var(--onyx-number-size-100);
+  --onyx-size-border-lg: var(--onyx-number-size-150);
+  --onyx-opacity-full: var(--onyx-number-opacity-100);
+  --onyx-opacity-90: var(--onyx-number-opacity-200);
+  --onyx-opacity-80: var(--onyx-number-opacity-300);
+  --onyx-opacity-70: var(--onyx-number-opacity-400);
+  --onyx-component-navbar-text-default: var(--onyx-color-text-body-paragraph-intense);
+  --onyx-component-navbar-text-hover: var(--onyx-color-text-body-paragraph-intense);
+  --onyx-component-navbar-text-active: var(--onyx-color-text-body-link-default);
+  --onyx-component-navbar-text-selected: var(--onyx-color-text-body-link-default);
+  --onyx-opacity-60: var(--onyx-number-opacity-500);
+  --onyx-opacity-50: var(--onyx-number-opacity-600);
+  --onyx-opacity-40: var(--onyx-number-opacity-700);
+  --onyx-component-navbar-surface-default: var(--onyx-component-navbar-bg-blank);
+  --onyx-component-navbar-surface-hover: var(--onyx-color-surface-brand-primary-300);
+  --onyx-component-navbar-surface-active: var(--onyx-color-surface-brand-primary-300);
+  --onyx-component-navbar-surface-selected: var(--onyx-component-navbar-bg-blank);
+  --onyx-opacity-30: var(--onyx-number-opacity-800);
+  --onyx-component-navbar-border-default: var(--onyx-color-surface-brand-neutral-300);
+  --onyx-opacity-20: var(--onyx-number-opacity-900);
+  --onyx-opacity-10: var(--onyx-number-opacity-950);
+  --onyx-component-navbar-bg-tinted: var(--onyx-color-brand-neutral-1200);
+  --onyx-opacity-transparent: var(--onyx-number-opacity-1000);
+  --onyx-component-navbar-bg-blank: var(--onyx-color-brand-neutral-1100);
+  --onyx-component-button-primary-text-default: var(--onyx-color-text-grey-white);
```

Comment:
@jannick-ux Are all those button variables really needed? I though we decided to only use component tokens if they are really needed and most likely use semantic token :)

Personally I think this leads to huge number of tokens that might confuse the user

Reply:
basically they are needed, because the colors from light mode are not the parallel equivalent to the dark mode. In this case for the buttons they are necessary. But we can talk about it in detail next meeting (also on the restriction an access of the tokens to devs and designers..

Reply:
at least for the secondary button it is needed. I think 95% of the token systematic is fix but maybe I have to tweak some little spots.